src/org/ods/orchestration/util/Project.groovy

```diff
@@ -780,7 +780,7 @@ class Project {
 
     @NonCPS
     Map<String, DocumentHistory> getDocumentHistories() {
-        return this.data.documentHistories ?: [:]
+        return this.data.documentHistories
```

Comment:
Is this.data.documentHistories properly initialised?

Reply:
It is initialized to an empty map in the constructor and nobody touches it anywhere else.